lib/vanagon/platform/windows.rb

```diff
@@ -37,12 +37,19 @@ def package_name(project)
       # @param name [String] name of the project
       # @param binding [Binding] binding to use in evaluating the packaging templates
       def generate_packaging_artifacts(workdir, name, binding)
-        # nuget templates that do require a name change
-        erb_file(File.join(VANAGON_ROOT, "resources/windows/nuget/project.nuspec.erb"), File.join(workdir, "#{name}.nuspec"), false, { :binding => binding })
+        if @package_type == "nuget"
```

Comment:
:) I figured it was worth another look. We're adding a lot of wix specific variables, which would suck if we had to fail building nuget packages for all of them